### ISSUE TITLE
[Bug] Apple rejects UIWebView even though linker says references have been removed

### ISSUE THREAD

**Issue by jmichas** [xamarin/Xamarin.Forms#10554]
### Description
Ok, I'm still stuck. I've done some testing with my project and this is what I've found out.
If I run it with only `--warn-on-type-ref=UIKit.UIWebView -warnaserror:1503`
my build fails with the MT1503 which is the UIWebView is still present after linking.

If I run it with the warning plus `--optimize=experimental-xforms-product-type`
I get the same result, error on MT1503, so it seems that this does nothing.

If I run it with `--optimize=force-rejected-types-removal` my build succeeds with no message about anything UIWebView after linking. But Apple will still reject it with the ITMS-90809. So, I don't get what the problem is. I don't use UIWebView anywhere in my project.

I have used dotpeek and pulled in all of the dll's contained in my ipa and can't find a reference to UIWebView other than in the Xamarin.Forms.Platform.iOS. This occurs in my azure pipeline and builds that I have done locally and tried to upload with altool.

### Steps to Reproduce

1. 
2. 
3. 

### Expected Behavior
If the linker doesn't have MT1503 warning or error (depending on linker switch) I expect that Apple would not reject the build.

### Actual Behavior
Apple rejects with `ITMS-90809: Deprecated API Usage - New apps that use UIWebView are no longer accepted. Instead, use WKWebView for improved security and reliability. Learn more  `
### Basic Information

- Version with issue: 4.6
- Last known good version:
- IDE: Visual Studio 2019 or Azure Build Pipeline
- Platform Target Frameworks: 
  - iOS:  13.4
  - Android:  
  - UWP:   
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:
- IDE
Microsoft Visual Studio Enterprise 2019
Version 16.5.4
VisualStudio.16.Release/16.5.4+30011.22
Microsoft .NET Framework
Version 4.8.03761

Installed Version: Enterprise

Xamarin   16.5.000.533 (d16-5@9152e1b)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.
Xamarin.iOS and Xamarin.Mac SDK   13.16.0.13 (b75deaf)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.

### Workaround
None

**Comment by haiduong741** [comment]
what is your "Last known good version" ?

**Comment by jmichas** [comment]
So, my last good version was with XF 4.5 before April 30th. After the Apple deadline that version stopped working.

**Comment by nbevans** [comment]
Apple isn't actually enforcing this yet until end of Q4 2020. I just submitted an app today with this warning being emitted. Seemed to go through fine...

**Comment by jmichas** [comment]
Ok, so I figured out my problem.
It turns out that Google MobileAds AdMob nugets were somehow adding in code that the linker didn't catch. When I opened my ipa and used terminal to `grep -r UIWebView .` I found this
```
Binary file ./Frameworks/PersonalizedAdConsent.framework/PersonalizedAdConsent matches
```
I was only one version behind on firebase and google mobileads so I upgraded to the latest and got the build to work and Apple accepted it.

I am still using the `--optimize=force-rejected-types-removal --warn-on-type-ref=UIKit.UIWebView -warnaserror:1503` mtouch args though.
